client/src/components/squids/SquidForm.jsx

```diff
@@ -0,0 +1,120 @@
+import React, { useState } from "react";
+
+import "../../style/squids/squidForm.pcss";
+
+import { useForm } from "react-hook-form";
+
+import { ApiClient } from "../../backend/ApiClient";
+import { FormError } from "../common/FormError";
+
+export const SquidForm = ({ setRefreshMessage }) => {
+  const {
+    handleSubmit,
+    register,
+    formState: { errors },
+    setError,
+  } = useForm({
+    defaultValues: {
+      experiencePoints: 0,
+    },
+  });
+
+  const onSubmit = async (data) => {
+    const postSquidResponse = await ApiClient.post("/squids", {
+      ...data,
+    });
+
+    if (!postSquidResponse.errors) {
+      setRefreshMessage(true);
+    } else {
+      // translateServerErrors(postSquidResponse.errors, setError);
```

Comment:
You'll want to do any conditional logic that's based on your server response within the onSuccess/onError/onSettled/onMutation hooks provided by React Query useMutation, once you have that implemented :)